### DISCUSSION TITLE
Github 这是出啥问题了么。。。releases 里面默认是去年的版本

### DISCUSSION THREAD

**Discussion by breakstring** [lobehub/lobehub#4982] [General]
打开后都是好早的版本了，看不到现在的版本，但是从首页上右侧的链接点过去能到现在的 v1.36.3 

![image](https://github.com/user-attachments/assets/194b073f-370e-4309-9b8e-97cf7584165d)

**Comment by arvinxx** [comment]
排查了下，是我们的组织 bot 账号被 github flag 了。账号还能登录，数据都还在，但账号和活跃记录 public 不可见。

目前已经提交了恢复申请，等官方回复

**Comment by arvinxx** [comment]
目前看应该已经恢复了
